### PR TITLE
Restructuring project to compile in Android Studio

### PR DESCRIPTION
**Pull request by gscdev**

### CODE REVIEW

**Comment by jogrimst** [comment]
@gscdev To avoid including Android Studio files in versioning control, add a `.gitignore` file. See example at https://github.com/literacyapp-org/literacyapp-android/blob/master/.gitignore

**Comment by jogrimst** [comment]
@gscdev Did you delete the `app/build/` folder also?

**Comment by gscdev** [comment]
@jogrimst Yes, I deleted it.

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
+# Ignore Gradle GUI config
+gradle-app.setting
+
+# Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
+!gradle-wrapper.jar
+
+*.iml
+.idea
+
+*.mk
```
Comment (jogrimst) [inline .gitignore]: @gscdev Should we ignore `.mk` files? It seems like `Android.mk` is already included in the upstream repository at https://github.com/CyanogenMod/android_packages_apps_Calculator/blob/cm-13.0/Android.mk

**Pull request merged**